modules/va_forms/config/locales/en.yml

```diff
@@ -7,6 +7,7 @@ en:
       benefit_categories: 'Listing of categories and match'
       benefit_category_name: 'Name of the benefit category of the form'
       benefit_category_description: 'Description of the benefit category of the form'
+      deleted_at: "The timestamp at which the form was deleted"
```

Comment:
2 questions for this one:
1. is this a good enough description?
2. assuming it is, do I have to do anything to initiate translations of it?